### PR TITLE
[Bug #1533] Ensure vocabularies are loaded for imported vocabulary list edit

### PR DESCRIPTION
**Pull request by ledsoft**
https://kbss.felk.cvut.cz/redmine/issues/1533

### CODE REVIEW

File: src/component/vocabulary/ImportedVocabulariesListEdit.tsx [hunk 1]
```diff
@@ -10,16 +10,25 @@ import {connect} from "react-redux";
...
     vocabularies: { [key: string]: Vocabulary };
     importedVocabularies?: AssetData[];
     onChange: (change: object) => void;
+    loadVocabularies: () => void;
 }
 
 export class ImportedVocabulariesListEdit extends React.Component<ImportedVocabulariesListEditProps> {
 
+    public componentDidMount() {
+        if (Object.getOwnPropertyNames(this.props.vocabularies).length === 0) {
```
Comment (psiotwo) [inline src/component/vocabulary/ImportedVocabulariesListEdit.tsx:27]: won't this cycle on fresh deploys?

**Review by psiotwo — COMMENTED** [review]
see comment

File: src/component/vocabulary/ImportedVocabulariesListEdit.tsx (hunk 1 above)
Comment (ledsoft) [reply src/component/vocabulary/ImportedVocabulariesListEdit.tsx:27]: Why? it is invoked only in didmount.

Comment (psiotwo) [reply src/component/vocabulary/ImportedVocabulariesListEdit.tsx:27]: true

**Review by psiotwo — APPROVED** [review]
lgtm

**Pull request merged**